docs/CONTRIBUTING.md

```diff
@@ -2,60 +2,67 @@
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 # Contributing
 
-- [Pull Requests](#pull-requests)
-  - [Commits](#commits)
 - [Requirements](#requirements)
-- [Development Workflow](#development-workflow)
-  - [Updating VS Code](#updating-vs-code)
-    - [Notes about Changes](#notes-about-changes)
-- [Build](#build)
-- [Testing](#testing)
-  - [Unit Tests](#unit-tests)
+- [Creating pull requests](#creating-pull-requests)
+  - [Commits and commit history](#commits-and-commit-history)
+- [Development workflow](#development-workflow)
+  - [Update VS Code](#update-vs-code)
+  - [Build](#build)
+  - [Test](#test)
+  - [Unit tests](#unit-tests)
   - [Integration Tests](#integration-tests)
-  - [End-to-End Tests](#end-to-end-tests)
+  - [End-to-end tests](#end-to-end-tests)
```

Comment:
nit: If "Integration Tests" is title-cased, shouldn't "End-to-end tests" be too? i.e. "End-to-end Tests"